host/src/verify_blob.rs

```diff
@@ -144,7 +131,7 @@ impl From<BlobVerificationProof> for common::blob_info::BlobVerificationProof {
 
 pub fn decode_blob_info(
     inclusion_data: Vec<u8>,
-) -> Result<(BlobHeader, BlobVerificationProof), anyhow::Error> {
+) -> Result<(BlobHeader, BlobVerificationProof, Vec<u8>), anyhow::Error> {
```

Comment:
Please document what this function does and its outputs (specially the last element from the tuple)